modules/test/playwright/helpers/HeadlessAdminTaxonomyApiHelper.ts

```diff
@@ -85,6 +91,24 @@ export class HeadlessAdminTaxonomyApiHelper {
 		);
 	}
 
+	/**
+	 * It allows creating a subcategory inside a category.
+	 *
+	 * @param name the name of the subcategory
+	 * @param vocabularyId the parent vocabulary id
+	 */
+
+	async postTaxonomyCategoryTaxonomyCategory({
```

Comment:
A little doubt here, this shouldn't be post...Subcategory instead of post...Category?